installation.mdx

```diff
@@ -63,21 +71,24 @@ pnpm dlx mint dev
 
 </CodeGroup>
 
-A local preview of your documentation will be available at `http://localhost:3000`.
-
 ## Updates
 
-Each CLI release is associated with a specific version of the Mintlify platform. If your local preview doesn't align with the production version, please update the CLI:
+Each CLI release is associated with a specific version of the Mintlify platform. If your local preview is out of sync with the production version, please update the CLI:
```

Comment:
I think it might be worth calling out that there are 2 versions.

There is the CLI version and the client version.

The CLI version can be updated by running `npm i mint@latest -g`.

The client version can be updated by running `mint update`.

The CLI version will bump if we change any of the functions that are ran in the CLI - this probably will happen decently frequently but not as frequently as you will need to update the client which are the frontend changes.

![CleanShot 2025-06-12 at 21 36 36@2x](https://github.com/user-attachments/assets/86f7fe2b-6bf5-41e3-9daa-da9ba1ee2bb0)